src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx

```diff
@@ -18,17 +18,35 @@ redirects:
   - /docs/mobile-monitoring/new-relic-mobile-ios/install-configure/upgrade-new-relic-mobiles-ios-sdk-v4
   - /docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk-v4/
 ---
+ For information about the latest version, refer to the [release notes](/docs/releases/ios).
 
-You must be an account Admin to install, configure, and upgrade the iOS agent. For information about the latest version, refer to the [release notes](/docs/releases/ios).
+## Update with Swift Package Manager [#SPM]
 
-## Replace your iOS framework [#upgrading]
+Updating the New Relic iOS agent using Swift Package Manager:
 
-First, you'll need to replace the earlier version of your iOS agent framework:
+1. Navigate to the NewRelic agent in the Package Dependencies section of the xCode project navigator.
+2. Right-click on the **NewRelic** agent.
+3. Select **Update Package**.
 
-1. From the Project Navigator (**CMD 1**) in Xcode, search for **NewRelic.xcframework**.
-2. Right-click or control-click **NewRelic.xcframework**, and select **Show in Finder**.
-3. Drag **NewRelic.xcframework** to the trash.
-4. Verify that the Xcode project highlights the reference to **NewRelic.xcframework** in red.
-5. Right-click or control-click **NewRelic.xcframework**, and select **Delete** to remove the obsolete reference from the project.
 
-Then, continue with the installation process in the UI at **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile > (select an app) > Settings > Installation**.
+## Update with Cocoa Pods [#Cocoa-Pods]
+
+Updating the New Relic iOS agent using Cocoa Pods:
+
+1. Open the **Terminal** app and navigate to the directory with your PodFile.
+2. Run the following command:
+```
+pod update NewRelicAgent
+```
+
+
+## Manually replace your iOS framework [#Manual]
+
+Downloading and manually installing the New Relic iOS agent.
+
+1. First click this [link](https://download.newrelic.com/ios_agent/) to be taken to our iOS agent hosting site.
+2. Click the version of the iOS agent you want to update to. This will begin the download.
```

Comment:
```suggestion
```suggestion
1. Go to https://download.newrelic.com/ios_agent/ and select the iOS agent you want to update to. A download will be initiated. 
```
```